src/www/ui/scripts/change-license-common.js

```diff
@@ -264,6 +264,13 @@ function openTextModel(uploadTreeId, licenseId, what, type) {
     $('#selectFromNoticeFile').css('display','none');
   }
 
+  if (what == 2 || what === 'reportinfo') {
+    // clicked to add button to display child modal
+    $('#clearText').css('display','inline-block');
+  } else {
+    $('#clearText').css('display','none');
```

Comment:
Please consider using [$.show()](https://api.jquery.com/show/) and [$.hide()](https://api.jquery.com/hide/)